components/components-styles/badge.css

```diff
@@ -0,0 +1,79 @@
+:root{
+    --PRIMARY-BADGE-COLOR: #4B4453;
+    --BADGE-TEXT-COLOR :#FFF;
+    --STATUS-ONLINE-COLOR:#ff9204;
+    --STATUS-OFFLINE-COLOR: #808080;
+    --STATUS-DND-COLOR: #FF0000;
+}
+
+.badge-container{
+    position: relative;
+}
+
+.badge{
+    position: absolute;
+    padding: .4em 1.2em;
+    background-color: var(--PRIMARY-BADGE-COLOR);
+    color: var(--BADGE-TEXT-COLOR);
+    font-size: .75em;
+    letter-spacing: .1em;
+    text-align: center;
+    border-radius: 5px;
+    margin-left:1rem;
+}
+
+.badge-icon{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 2.5rem;
+    height: 2.5rem;
+    border-radius: 50%;
+    text-align: center;
+    font-size: 1.6rem;
+    padding: 0;
+    right: -15px;
+    top:-5px;
```

Comment:
Convert px to rem values. The rest of the code looks good.

Reply:
> Convert px to rem values. The rest of the code looks good.

Hi, I have made the above changes.